docs/ext/pydantic_autosummary/templates/autosummary/pydantic_model.rst

```diff
@@ -0,0 +1,51 @@
+{{ name | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autosummary::
+   :toctree: {{ name }}
+   :recursive:
+
+   .. toctree::
+      :maxdepth: -1
+
+   {% block model_fields %}
+   {% for field in model_fields %}
+      ~{{ field }}
+   {% endfor %}
+   {% endblock %}
+
+.. autopydantic_model:: {{ objname }}
+   :members:
+   :inherited-members: BaseModel
+   :model-show-config-summary: False
+   :model-show-json: False
+   :model-show-validator-members: False
+   :model-show-validator-summary: False
+   :field-list-validators: False
```

Comment:
These options descriptions are [documented here](https://autodoc-pydantic.readthedocs.io/en/stable/users/configuration.html#model). If we make a parallel model documentation for ourselves, some of these may want to be removed, hence they were not set in conf.py instead.